build-support/travis/travis.yml.mustache

```diff
@@ -10,7 +10,9 @@ conditions: v1
 env:
   global:
     - PANTS_CONFIG_FILES="${TRAVIS_BUILD_DIR}/pants.travis-ci.ini"
-    - PYTEST_PASSTHRU_ARGS="-v --duration=3"
+    # With Pytest, we only want to output failures and skips.
```

Comment:
IIRC env changes thrash through Travis cache. Hiw about taking the opportunity to inline these values in ci.sh as the default values for the env var (or maybe just inline fully).